### PR TITLE
Update createNavMesh.md

### PR DESCRIPTION
**Pull request by yar2001**
Hi, thank you for your amazing work! It seems possible to add some additional information about the parameters here

### CODE REVIEW

**Review by CedricGuillemet — CHANGES_REQUESTED** [review]
http://recastnavigation.org/ is inaccessible

**Review by CedricGuillemet — APPROVED** [review]

File: content/features/featuresDeepDive/crowdNavigation/createNavMesh.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ detailSampleDist - Sets the sampling distance to use when generating the detail
 
 detailSampleMaxError - The maximum distance the detail mesh surface should deviate from heightfield data. World Units.
 
+You can find more information about those parameters on the [Recast documentation](http://recastnav.com/structrcConfig.html).
```
Comment (sebavan) [inline content/features/featuresDeepDive/crowdNavigation/createNavMesh.md]:
```suggestion
You can find more information about those parameters on the [Recast documentation](https://recastnav.com/structrcConfig.html).

```

File: content/features/featuresDeepDive/crowdNavigation/createNavMesh.md [hunk 2]
```diff
@@ -86,6 +86,9 @@ detailSampleDist - Sets the sampling distance to use when generating the detail
 
 detailSampleMaxError - The maximum distance the detail mesh surface should deviate from heightfield data. World Units.
 
+You can find more information about those parameters on the [Recast documentation](https://recastnav.com/structrcConfig.html).
```
Comment (sebavan) [inline content/features/featuresDeepDive/crowdNavigation/createNavMesh.md:89]:
```suggestion
You can find more information about those parameters on the [Recast documentation](https://recastnav.com/structrcConfig.html).
```

**Review by sebavan — APPROVED** [review]
Can you

**Pull request merged**